### ISSUE TITLE
eos_l2_interfaces does not set switchport mode

### ISSUE THREAD

**Issue by acalm** [ansible-collections/eos#14]
##### SUMMARY

eos_l2_interfaces does not set switchport mode when configuring acces/trunk, the module does not accept the mode parameter as checked for here: https://github.com/ansible-collections/eos/blob/71b8b03c86d1dc15f2ee3091a9a08080b47a3621/plugins/module_utils/network/eos/config/l2_interfaces/l2_interfaces.py#L220

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

eos_l2_interfaces

##### ANSIBLE VERSION

```paste below
ansible 2.10.0.dev0
  config file = /home/acalm/.ansible.cfg
  configured module search path = ['/home/acalm/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/acalm/repos/ansible/lib/ansible
  executable location = /home/acalm/repos/ansible/bin/ansible
  python version = 3.6.10 (default, Feb 27 2020, 10:55:54) [GCC 9.2.0]
```

##### CONFIGURATION

```paste below
COLOR_VERBOSE(/home/acalm/.ansible.cfg) = magenta
INVENTORY_IGNORE_EXTS(/home/acalm/.ansible.cfg) = ['.pyc', '.pyo', '.swp', '.bak', '~', '.rpm', '.md', '.txt', '~', '.orig', '.ini', '.cfg', '.retry']
RETRY_FILES_ENABLED(/home/acalm/.ansible.cfg) = False
```

##### OS / ENVIRONMENT

centos, rhel 7

##### STEPS TO REPRODUCE

```yaml
- name: Configure trunk on Ethernet7
  eos_l2_interfaces:
    config:
      - name: Ethernet37
        trunk:
          native_vlan: 67
    state: replaced
```

##### EXPECTED RESULTS

```
interface Ethernet37
   switchport trunk native vlan 67
   switchport mode trunk
```
##### ACTUAL RESULTS

```
interface Ethernet37
   switchport trunk native vlan 67
```

```paste below

```

**Comment by acalm** [comment]
mixed up versions, this has already been fixed (d8c5c6eb9deb7c3f8fef81fbf88899a7c2cf8638)

**Comment by sc68cal** [comment]
I just came across this in #60 - this needs to be backported
